portal/migrations/0002_host_os.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('portal', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='host',
+            name='os',
+            field=models.CharField(max_length=32, null=True, choices=[(b'routeros', b'RouterOS'), (b'airos', b'AirOS'), (b'linux', b'Linux'), (b'esxi', b'ESXi'), (b'windows', b'Windows'), (b'ilo', b'iLO'), (b'other', b'Other'), (None, b'None')]),
+            preserve_default=True,
+        ),
+    ]
```

Comment:
Following the docs, you should be able to find an example of how to set initial values for the routers somewhere around this part of the migration.

Reply:
Here's the command:

UPDATE portal_host SET os='routeros' WHERE type IN ('sector', 'ptp', 'edgerouter', 'cellrouter', 'client');

Please provide Django translation and I'll update PR.

Reply:
Check out https://docs.djangoproject.com/en/1.8/topics/migrations/#data-migrations. You will have to write some python.

Reply:
Nah, I'm not gonna waste time re-expressing perfectly good SQL in Python.  My time is valuable, and this isn't even the main quest.  Take it or leave it.

Reply:
I'm sure there is a way to pass raw SQL through the ORM in the migration, but since that's not the normal way of doing it I can't offer any expertise.

I noticed the SQL you provided is not idempotent if you happened to run the migration again after manually managing some records. I'd recommend introducing a WHERE NOT NULL clause, regardless of which language you write the migration in.

Reply:
It's ephemeral SQL, I'm not worried about idempotency.  Maybe at some point later you can share the expert idiomatic way so I can learn it for any future changes.

Thanks, I think this is all the value I'm gonna get out of this review.  Pushing it in and moving on.